### PR TITLE
SO-877 - buyer guide link; remove empty field msg

### PR DESCRIPTION
**Pull request by adohall**
Note - the empty field message appeared on page load; it is now commented out; field is marked as mandatory and user can't submit without entering a password, so message isn't really needed

### CODE REVIEW

File: app/templates/signup.hbs [hunk 1]
```diff
@@ -22,7 +22,7 @@
       <section class={{if (eq accountType "buyer") "target"}}>
         <h3>Create a buyer account</h3>
         <p><b>You can sign up as a buyer if you're a NSW Government employee or an employee of another eligible organisation.</b></p>
-        <p>To get started, register your email with us. Once we confirm your email, you will be able to view supplier profiles and products. For more information see the buyer guide.</p>
+        <p>To get started, register your email with us. Once we confirm your email, you will be able to view supplier profiles and products. For more information see the {{ link-to "buyer guide" "buyer-guide" }}.</p>
```
Comment (armansa) [inline app/templates/signup.hbs:25]: Is the buyer-guide page still maintained? I think we replaced it with a page in guided buying.

Comment (adohall) [reply app/templates/signup.hbs:25]: There's also a link to this buyer guide from the 'contact page. I'll ask Lucy & team.

Comment (armansa) [reply app/templates/signup.hbs:25]: I created a ticket for it: https://buynsw.atlassian.net/browse/SO-956

Comment (adohall) [reply app/templates/signup.hbs:25]: Ok. In the meantime I'll change this link here and in contact page, as per Dan's request.

**Pull request merged**